src/components/footer/templates/global-footer-section.handlebars

```diff
@@ -1,6 +1,12 @@
 <section class="wds-global-footer__{{parentName}}-section wds-is-{{name}}">
     {{#if model.header.title}}
-        <h3 class="wds-global-footer__section-header">{{i18n model.header.title.key ns='design-system'}}</h3>
+        <h3 class="wds-global-footer__section-header">
+            {{#if (equal 'fandom-overview' name)}}
```

Comment:
is this in case someone else is using this package?

Reply:
Not intentionally. It was the most obvious solution for selecting that specific footer section to me, is there a better way to do this?

Reply:
Oh, you're right. Took me a sec to remind myself how this all worked. Carry on 👍